src/accordion/accordion.ts

```diff
@@ -116,7 +116,8 @@ export interface NgbPanelChangeEvent {
       <div class="card">
         <div role="tab" id="{{panel.id}}-header" [class]="'card-header ' + (panel.type ? 'bg-'+panel.type: type ? 'bg-'+type : '')">
           <h5 class="mb-0">
-            <button class="btn btn-link" (click)="!!toggle(panel.id)" [disabled]="panel.disabled" [class.collapsed]="!panel.isOpen"
+            <button type="button" class="btn btn-link"
+              (click)="!!toggle(panel.id)" [disabled]="panel.disabled" [class.collapsed]="!panel.isOpen"
```

Comment:
You could remove the `!!` here as it is not needed anymore. It used to be needed when the `<button>` was an `<a>`.

Returning `false` to prevent default only work on `<a>` tag (and it is not standard, and should not be used anymore BTW).

On a button it is effect-less. Main issue here was definitely only related to the missing type.

Reply:
Good point @benouat - thnx, removed it.